bundles/core/src/main/java/com/adobe/cq/commerce/core/cacheinvalidation/internal/InvalidateCacheButtonServlet.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ *
+ *    Copyright 2025 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+package com.adobe.cq.commerce.core.cacheinvalidation.internal;
+
+import javax.annotation.Nonnull;
+import javax.servlet.Servlet;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.adobe.granite.ui.components.rendercondition.RenderCondition;
+import com.adobe.granite.ui.components.rendercondition.SimpleRenderCondition;
+
+@Component(
+    service = { Servlet.class },
+    property = {
+        "sling.servlet.resourceTypes=" + InvalidateCacheButtonServlet.RESOURCE_TYPE,
+        "sling.servlet.methods=GET",
+        "sling.servlet.extensions=html"
+    })
+public class InvalidateCacheButtonServlet extends SlingSafeMethodsServlet {
+
+    public static final String RESOURCE_TYPE = "core/cif/components/renderconditions/dispatcherConfigCondition";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(InvalidateCacheButtonServlet.class);
+
+    @Reference(cardinality = ReferenceCardinality.OPTIONAL, policyOption = ReferencePolicyOption.GREEDY)
+    private transient InvalidateCacheSupport invalidateCacheSupport;
+
+    @Override
+    protected void doGet(@Nonnull SlingHttpServletRequest request, @Nonnull SlingHttpServletResponse response) {
+        boolean conditionMet = invalidateCacheSupport != null;
+        if (!conditionMet) {
+            LOGGER.error("InvalidateCacheSupport service is not available");
```

Comment:
Avoid logging error here. This is normal if a customer is not using this feature.